api/src/main/java/com/vimeo/networking2/adapters/ErrorHandlingCallAdapter.kt

```diff
@@ -0,0 +1,33 @@
+package com.vimeo.networking2.adapters
+
+import com.vimeo.networking2.ApiError
+import okhttp3.ResponseBody
+import retrofit2.Call
+import retrofit2.CallAdapter
+import retrofit2.Converter
+import java.lang.reflect.Type
+import java.util.concurrent.Executor
+
+/**
+ * Custom call adapter to handle errors.
+ *
+ * @param responseType              Type of the response.
+ * @param callbackExecutor          The executor upon which the request is made.
```

Comment:
Is this documentation correct? Seems like it should be the executor on which the caller is notified.